bootstrap/manifest_template.json

```diff
@@ -515,6 +515,11 @@
     "url": "http://status-web.discoverd"
   },
```

Comment:
Yeah you are right. I tested the actual status check by adding a sleep to the start of the status-check step to give me time to go break something. But if it's broken at that point it does indeed not pass.